hibernate-reactive-core/src/main/java/org/hibernate/reactive/context/impl/VertxContext.java

```diff
@@ -59,22 +64,29 @@ public <T> T get(Key<T> key) {
 	public void remove(Key<?> key) {
 		final io.vertx.core.Context context = Vertx.currentContext();
 		if ( context != null ) {
-			context.removeLocal( key );
+			boolean removed = context.removeLocal( key );
+			LOG.tracef( "Key %s removed from context: %s", key, removed );
```

Comment:
are you sure about using "%s" for a boolean primitive?

Reply:
Yes, you can use it with any type